catalogue/webapp/components/WorkDetailsText/WorkDetailsText.tsx

```diff
@@ -9,6 +9,7 @@ const LimitWidth = styled.div.attrs({
   className: 'spaced-text',
 })`
   max-width: 45em;
+  font-variant-ligatures: no-contextual;
```

Comment:
Let's add a comment with an example of what sort of ligatures we're trying to prevent.